### PR TITLE
sync vs async example v1

### PR DESCRIPTION
**Pull request by Pierox57**

### CODE REVIEW

**Comment by AMontagu** [comment]
Thank you for the work.

It will need informations about the fact that we plan to remove Sync support in V1.0.0 and explation about the risk of using stream in sync mode as it block the all thread while not finished. 

It would be great to also document all the method that are async ready by DSG (ais_valid, amessage, aget_serializer, ...)

I am available if you need more informations from me to do that

**Comment by legau** [comment]
You're using the old doc, you have to merge from the `rtd` branch.

File: docs/inner-workings/sync-vs-async.rst [hunk 1]
```diff
@@ -1,4 +1,114 @@
...
+     - ``django_socio_grpc.generics.ListCreateService``
+   * - ``django_socio_grpc.generics.AsyncReadOnlyModelService``
+     - ``django_socio_grpc.generics.ReadOnlyModelService``
+   * - ``django_socio_grpc.generics.AsyncModelService``
+     - ``django_socio_grpc.generics.class ModelService``
+
+Async ready methods
+-------------------
+
+Since Django 4.1, Django can run ORM queries asynchronously.
```
Comment (AMontagu) [inline docs/inner-workings/sync-vs-async.rst:82]: ORM is stil synchrone. It's the view that can bu run asynchronously

File: docs/inner-workings/sync-vs-async.rst [hunk 2]
```diff
@@ -1,4 +1,114 @@
...
+    * ais_valid
+    * acreate
+    * aupdate
+    * adata
+    * amessage
+
+Sync support
+------------
+
+If you want to use the sync mode, you should know that we will no longer support this mode from version 1.0.0 of Django Socio gRPC.
```
Comment (AMontagu) [inline docs/inner-workings/sync-vs-async.rst:114]:
```suggestion
If you want to use the sync mode, you should know that we are thinking about droping sync support from version 1.0.0 of Django Socio gRPC.
```

**Pull request closed without merging**